src/systemctl/systemctl.c

```diff
@@ -8650,6 +8650,7 @@ static int logind_cancel_shutdown(void) {
 int main(int argc, char*argv[]) {
         _cleanup_(pager_closep) Pager pager;
         _cleanup_(polkit_agent_closep) PolkitAgent agent;
+        _cleanup_(ask_password_agent_closep) AskPasswordAgent agent2;
```

Comment:
btw, I wonder if we stick with this approach, whether doing a macro for each wouldn't be nicer, i.e. `DESTRUCT_PAGER;`, `DESTRUCT_POLKIT_AGENT;` and so on, which declares the variable properly under a unique name